### ISSUE TITLE
pre-trained resnext model doesn't work

### ISSUE THREAD

**Issue by coolbrain** [apache/incubator-mxnet#6682]
I  downloaded the pre-trained resnext-50 model from http://data.mxnet.io/models/imagenet/resnext/50-layers/  and I processed the imagenet1k_val data by 
python ${CUR_DIR}/../../../tools/im2rec.py --resize 256 --quality 90 --num-thread 16 imagenet1k-val val/
But when I execute this script in the terminal: 
python score.py  --gpus 0,1,2,3 --model=imagenet1k-resnext-50 --data-val=../../../../data/imagenet1k-val.rec --data-nthreads 16
print:
INFO:root:Finished with 438.658564 images per second
INFO:root:('accuracy', 7.992327365728901e-05)
INFO:root:('top_k_accuracy_5', 0.0021379475703324807)

The above print is obvious wrong. How this problem come out ?
I try other pre-trained models such as resnet-50, they don't have the above problem.

**Comment by zihaolucky** [comment]
Have you checked http://mxnet.io/how_to/finetune.html ?

**Comment by coolbrain** [comment]
@zihaolucky Yes，I have read it. it works in pre-trained resnet model, but can't work in pre-trained resnext model.  I try to check whether the normalization of image influence the result.

**Comment by huangzehao** [comment]
@coolbrain Hi, I also met this problem. I have test resnet50 and resnext50 model from mxnet model zoo.  The top-1 acc of resnet50 is 75.28 while resnext is the same as your result.
I also test resnext101, the acc is 9.09904e-5.

**Comment by szha** [comment]
@Jerryzcn

**Comment by winstywang** [comment]
@terrychenism

**Comment by terrychenism** [comment]
@coolbrain Did you use this list (http://data.mxnet.io/models/imagenet/resnext/val.lst) to create rec? Cuz we use different imglist to train resnext 50/101. Maybe I will update the model recently.

**Comment by coolbrain** [comment]
@terrychenism  I used this list([http://data.mxnet.io/models/imagenet/resnet/val.lst](url)),  would this influence the result?

**Comment by terrychenism** [comment]
@coolbrain 
The resnet val list is different with resnext. 

Here's the training log for resnext 50/101 https://gist.github.com/terrychenism/689db43abd0c21c365d26b7e17023e38
https://gist.github.com/terrychenism/9aa9732536de5d615072ef7fe38beb9d
You can use them as reference for training your own model.

**Comment by Jerryzcn** [comment]
For resnext101-64x4d use --rgb-mean 123.68,116.779,103.939.
I am not sure about resnext101-32x4d and resnext50-32x4d though

**Comment by terrychenism** [comment]
For resnext101-32x4d and resnext50-32x4d we don't need to do mean subtraction, cuz batch norm op after input data.  plz set --rgb-mean 0,0,0

**Comment by coolbrain** [comment]
@terrychenism  I trained the resnext model according to the training log you shared.  You first use 480 shorter side of image to train the model which can add more training data compared to 256, after 100 epochs, you use the 256 shorter side to  improve accuracy.  The top-1 of resnext-50 is 76.8%. But the top-1 of resnext-50 is 77.8% in Resnext paper implemented in Torch and the paper says they just use shorter side 256.  Can Mxnet has performance as good as  Torch?

**Comment by terrychenism** [comment]
@coolbrain Yes, we can reproduce resnext-101 result 78.8 using imageiter and fb-style augmentation (https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/image/image.py#L821)

**Comment by szha** [comment]
Feel free to reopen if the issue persists.

**Comment by hubenjm** [comment]
@terrychenism I wanted to ask if you could provide other parameters used in your image iterators for training the resnext-50 model in particular. e.g. what was the min and max random crop size?
